Makefile

```diff
@@ -264,6 +269,10 @@ verify-conversions: $(CONVERSION_VERIFIER)  ## Verifies expected API conversion
 verify-tiltfile: ## Verify Tiltfile format.
 	./hack/verify-starlark.sh
 
+.PHONY: verify-codespell
+verify-codespell: codespell ## Verify codespell.
+	@$(CODESPELL) $(ROOT_DIR) --ignore-words=$(ROOT_DIR)/.codespellignore --skip="*.git,*_artifacts,*.sum,$(ROOT_DIR)/hack"
```

Comment:
sorry, one last question: why did you add the hack dir here? I see in the github action that it only had `.git,_artifacts,*.sum` as skip setting

Reply:
Good question, sorry, I should have specified in the PR description about it.
I added `$(ROOT_DIR)/hack` to avoid codespell errors upon checking `cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/tests/test_basic.py`
Removing `$(ROOT_DIR)/hack` from the `verify-codespell` target leads to a bunch of errors like below. 

```
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:54: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:55: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:56: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:57: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:58: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:59: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:60: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:61: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:62: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:63: ECT ==> ETC
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:68: whic ==> which
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:72: ect ==> etc
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:73: infinit ==> infinite
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:74: infinit ==> infinite
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:76: ISNT ==> ISN'T
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:77: Teh ==> The
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:78: Teh ==> The
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:79: Teh ==> The
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:80: Teh ==> The
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:81: pallete ==> palette
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:82: pallete ==> palette
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:83: pallete ==> palette
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:84: virtualY ==> virtually
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:85: archType ==> archetype
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:86: Satelite ==> Satellite
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/linux-kernel.exclude:87: SMOE ==> SOME
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/dictionary_usage.txt:9: useability ==> usability
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell_lib/data/dictionary_usage.txt:17: useability ==> usability
/Users/nawazhussain/msftcode/cluster-api-provider-azure/hack/tools/bin/codespell_dist/codespell-2.2.5.dist-info/METADATA:55: adn ==> and
```

Reply:
But now that I think about it, I can further fine-tune just to remove codespell_dist folder. Updating.

Reply:
Updated! :)